pkg/common/logger.go

```diff
@@ -0,0 +1,43 @@
+package common
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// Wrap common output functions for testability
+type Logger interface {
```

Comment:
Sure. I have used logrus in the past and would use it for a non-prototype. It doesn't expose an interface though, but I am now reading that it has some testability features I wasn't aware of that would allow me to avoid using a mock. Generally, I would not be testing log output, but since this is a CLI application, it seemed like the behavior under test.